lib/app.ex

```diff
@@ -1,7 +1,12 @@
 defmodule App do
   @my_spotify_user_id Application.compile_env!(:roule_avec_spot, [Spotify, :user_id])
+
+  def create_playlist(video_id) do
+    create_playlist(video_id, nil)
+  end
+
   def create_playlist(video_id, playlist_name) do
```

Comment:
Just FYI, this can be written using a default arg:
```suggestion
  def create_playlist(video_id, playlist_name \\ nil) do
```